lib/cWarnings.mli

```diff
@@ -56,6 +56,10 @@ val create : name:string -> ?category:category -> ?default:status ->
   ('a -> Pp.t) -> ?loc:Loc.t -> 'a -> unit
 (** Combined creation function. [name] must be a fresh name. *)
 
+val create_with_quickfix : name:string -> ?category:category -> ?default:status ->
+    ('a -> Pp.t) -> ?loc:Loc.t -> ?quickfix:Quickfix.t list -> 'a -> unit
```

Comment:
An alternative API could be something like
```suggestion
val create : name:string -> ?category:category -> ?default:status ->
  ?quickfix:('a -> Quickfix.t list) ->
  ('a -> Pp.t) -> ?loc:Loc.t -> 'a -> unit
```
(since the quickfix generator is optional we don't need a separate create_with_quickfix)

Reply:
Sorry @SkySkimmer I though you wanted this to avoid some spurious lines in the patch that were eliding the optional argument after declaring the warning. Did I get it wrong?